lib/index.js

```diff
@@ -53,9 +53,11 @@ GA.prototype.screen = proxy('screen');
 
 function proxy(method){
   return function(message, fn){
-    if (this.settings.serversideClassic) {
-      return method === 'screen' ? setImmediate(fn) : this.classic[method].call(this.classic, message, fn);
+    var ctx = this.settings.serversideClassic ? this.classic : this.universal;
+    if ('function' === typeof ctx[method]) {
+      return ctx[method](message, fn);
+    } else {
+      return setImmediate(fn);
```

Comment:
the logic is very slightly different now; before we could never call `setImmediate(fn)` if the user was using universal, whereas now we can.

were we doing "if classic and method is 'screen' then setImmediate(fn)" to express "classic doesn't implement screen"?